kolibri/core/assets/src/views/k-filter-textbox/index.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+
+  <div class="k-filter">
+    <ui-icon
+      icon="search"
+      class="k-filter-icon"
+      :ariaLabel="$tr('filter')"
+    />
+
+    <input
+      v-model.trim="model"
+      type="search"
+      class="k-filter-input"
+      :placeholder="placeholder"
+      :aria-label="placeholder"
+      :autofocus="autofocus"
+    >
+
+    <ui-icon-button
+      icon="clear"
+      color="black"
+      size="small"
+      class="k-filter-clear-button"
+      :class="model === '' ? '' : 'k-filter-clear-button-visible'"
+      :ariaLabel="$tr('clear')"
+      @click="model = ''"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  import uiIcon from 'keen-ui/src/UiIcon';
+  import uiIconButton from 'keen-ui/src/UiIconButton';
+  /**
+   * Used to filter items via text input
+   */
+  export default {
+    name: 'k-filter-textbox',
```

Comment:
`Kolibri: Name id "k-filter-textbox" should be in camelCase. Found in /Users/d/Projects/learning_equality/repos/kolibri/kolibri/core/assets/src/views/k-filter-textbox/index.vue`

Reply:
But none of the component names are camelCase...

Reply:
Hm, not sure what that warning is referring to then

Reply:
Let's merge and open and issue for these lint warnings?

Reply:
okay that's fine

Reply:
These were the lint warnings you added to the extract_$trs plugin, @christianmemije.

Reply:
(Now that we are using component names as message namespaces).

Reply:
I assume that none of the other k-components have built in messages?

Reply:
That makes sense.

Reply:
Opened an issue regarding this: https://github.com/learningequality/kolibri/issues/2094